scopes/react/react/apps/web/plugins/prerender.ts

```diff
@@ -1,18 +1,10 @@
-import { join } from 'path';
-import PrerenderSPAPlugin from '@dreysolano/prerender-spa-plugin';
+import PrerenderSPAPlugin from 'prerender-spa-plugin-next';
+import { PuppeteerRenderer } from '@teambit/react.modules.prerenderer-puppeteer';
 import { ReactAppPrerenderOptions } from '../react-app-options';
 
-export const prerenderSPAPlugin = (prerender: ReactAppPrerenderOptions, staticDir: string) => {
+export const prerenderPlugin = (options: ReactAppPrerenderOptions) => {
   return new PrerenderSPAPlugin({
-    staticDir,
-    routes: prerender?.routes,
-    postProcess(renderedRoute: any) {
-      if (prerender.postProcess) return prerender.postProcess(renderedRoute, staticDir);
-      renderedRoute.outputPath = join(staticDir, `${renderedRoute.originalRoute}.html`);
-      return renderedRoute;
-    },
-    server: {
-      ...prerender?.server,
-    },
+    renderer: PuppeteerRenderer,
+    ...options,
```

Comment:
forwarding all options from the consumer.

- staticDir - is no longer needed with the new plugin, as it now respects the webpack output.path
- postProcess - the built in behavior works the same, and covers the '/' route
- server - forwarde
- renderer - using our custom renderer, with v14 support